core/controllers/base_test.py

```diff
@@ -323,13 +323,6 @@ def mock_get_current_time(unused_cls):
             self.assertFalse(base.CsrfTokenManager.is_csrf_token_valid(
                 'uid', token))
 
-    def test_redirect_oppia_test_server(self):
```

Comment:
@Jamesjay4199 any reason why this test was deleted? This decreased the coverage of the code-base and I am surprised how this got past the review cycle. I am currently looking how to make code-climate/codecov block PRs which decrease absolute coverage but in the mean time can you please revert this change ASAP? Thanks!

/cc @seanlip @brianrodri